src/components/exchange/ExchangeFloatingPanels.js

```diff
@@ -1,12 +1,19 @@
-import { FloatingPanels } from '../floating-panels';
-import styled from '@rainbow-me/styled-components';
+import React from 'react';
+import { Column } from '../layout';
 
-const ExchangeFloatingPanels = styled(FloatingPanels).attrs({
-  margin: 0,
-  translateY: ios ? 0 : -16,
-})({
-  paddingTop: 24,
-  ...(android && { height: '85%' }),
-});
+const ExchangeFloatingPanels = React.forwardRef(
```

Comment:
this was relying on the underlying floating panels component, I moved it into its own so its more independent

Reply:
don't mean to block but design system didn't help here?

Reply:
no dice